src/posit/connect/users_test.py

```diff
@@ -21,3 +23,79 @@ class TestUsers:
     def test_init(self, mock_config, mock_session):
         with pytest.raises(ValueError):
             Users(mock_config, mock_session, page_size=9999)
+
+    @responses.activate
+    def test_get_users(self):
+        responses.get(
+            "https://connect.example/__api__/v1/users",
```

Comment:
I made these fixtures by querying a real Connect and then sanitizing the responses.